libultraship/libultraship/Lib/Fast3D/gfx_pc.cpp

```diff
@@ -435,15 +436,15 @@ static void gfx_generate_cc(struct ColorCombiner *comb, uint64_t cc_id) {
                         val = SHADER_COMBINED;
                         break;
                     }
-                    // fallthrough for G_ACMUX_LOD_FRACTION
                     c[i][1][j] = G_CCMUX_LOD_FRACTION;
+                    [[fallthrough]]; // for G_ACMUX_LOD_FRACTION
```

Comment:
This syntax is black magic to me. I haven't seen it before. What is happening here and why did this file change?

Reply:
`[[attribute]]` is a standard way of specifying a c++ attribute.
`fallthrough` is a c++ standard attribute which, when specified, tells the compiler that a fallthrough in a case statement is intentional and thus should not result in a warning.